src/docs/community/faq.mdx

```diff
@@ -1,36 +1,115 @@
+import { Divider, FAQExpander } from '../../app/components';
+
 <!-- KEYWORDS: faq question answers -->
 
 # Frequently Asked Questions
 
-### 1. How do I update to PX Blue 2.2+ if I'm currently using 2.1?
+We strive to provide as much information as possible about PX Blue in our documentation as possible. However, we can't always cover everything. Below are some common questions that people have asked about the platform. Please check to see if your question has been answered below before contacting us directly.
+
+## Developers
+
+<FAQExpander question={`How do I start a project?`}>
```

Comment:
Is it possible to turn off the top divider in the first accordion item?

Reply:
It's weird...the MUI example doesn't have it on the first one...but their CSS rules suggest that it should be there.

Reply:
NVM, i figured it out. They had another rule to hide it for :first-child...but our Accordion is not the first child because it follows the header. Wrapping it in a div fixes it.